spec/support/helpers/service_helpers.rb

```diff
@@ -22,4 +22,27 @@ def mock_nomis_with(api, double_or_spy)
   def simulate_api_error_for(api, exception_class = Nomis::APIError)
     expect(Nomis::Api.instance).to receive(api).and_raise(exception_class)
   end
+
+  # allow feature tests to login for a specific prison
+  def prison_login(prison)
+    sso_response =
+      {
+        'uid' => '1234-1234-1234-1234',
+        'provider' => 'mojsso',
+        'info' => {
+          'first_name' => 'Joe',
+          'last_name' => 'Goldman',
+          'email' => 'joe@example.com',
+          'permissions' => [
+            { 'organisation' => prison.estate.sso_organisation_name, roles: [] }
+          ],
+          'links' => {
+            'profile' => 'http://example.com/profile',
+            'logout' => 'http://example.com/logout'
+          }
+        }
+      }
+
+    OmniAuth.config.add_mock(:mojsso, sso_response)
+  end
```

Comment:
Looks like this probably isn't needed, as it's unused and will need to be removed once HMPPS Auth is implemented by #2772 POM-807

Reply:
I've removed it, but it's a bit shocking that it isn't used. I think it was created for the new UI spikes (which have now been binned)